logger/main.go

```diff
@@ -2,102 +2,106 @@ package main
 
 import (
 	"flag"
-	"fmt"
 	"log"
 	"os"
 	"os/signal"
 	"strconv"
-	"strings"
 	"syscall"
 	"time"
 
-	"github.com/coreos/go-etcd/etcd"
-	"github.com/deis/deis/logger/syslogd"
+	"github.com/deis/deis/logger/configurer"
+	"github.com/deis/deis/logger/publisher"
+	"github.com/deis/deis/logger/storage"
+	"github.com/deis/deis/logger/syslogish"
+	"github.com/deis/deis/logger/weblog"
 )
 
 var (
 	logAddr         string
+	logHost         string
 	logPort         int
-	drainURI        string
 	enablePublish   bool
-	publishHost     string
-	publishPath     string
-	publishPort     string
+	webAddr         string
+	webPort         int
+	etcdHost        string
+	etcdPort        string
```

Comment:
could be avoided with the flag replacement but we've already established no external dependencies :)